src/easylink/step.py

```diff
@@ -1216,13 +1216,13 @@ class ParallelStep(TemplatedStep):
 
     @property
     def config_key(self):
-        """The pipeline specification key required for a ``ParallelStep``."""
-        return "parallel"
+        """The pipeline specification key required for a ``CloneableStep``."""
+        return "clones"
```

Comment:
This is the thing - it's what demands that the pipeline.yaml configuration key for parallel steps (i.e. CloneableSteps) is no longer "parallel" but rather "clones"